packages/elements-react/src/components/card/card-two-step.tsx

```diff
@@ -148,6 +151,17 @@ export function OryTwoStepCard() {
       return true
     })
 
+  // This is defined in Ory Kratos as well.
+  const noMethods: UiText = {
+    id: 5000002,
+    text: intl.formatMessage({
+      id: `identities.messages.${5000002}`,
```

Comment:
```suggestion
      id: `identities.messages.5000002`,
```

I found that react-intl is struggling to extract messages, if they're not plain strings.